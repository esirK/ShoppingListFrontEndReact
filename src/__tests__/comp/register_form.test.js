import React from 'react';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';


import RegisterForm from '../../components/register';

describe('Register Form Component', ()=>{
	const props ={
	};
	const wrapper = shallow(<RegisterForm {...props}/>);
	it('Renders Register form without clashing',()=>{
		expect(shallowToJson(wrapper)).toMatchSnapshot();
	});
});