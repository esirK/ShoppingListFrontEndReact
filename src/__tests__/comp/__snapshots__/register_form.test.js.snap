// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register Form Component Renders Register form without clashing 1`] = `
<Connect(Form(Connect(RegisterForm)))
  destroyOnUnmount={true}
  enableReinitialize={false}
  forceUnregisterOnUnmount={false}
  form="registrationform"
  getFormState={[Function]}
  keepDirtyOnReinitialize={false}
  persistentSubmitErrors={false}
  pure={true}
  shouldAsyncValidate={[Function]}
  shouldError={[Function]}
  shouldValidate={[Function]}
  shouldWarn={[Function]}
  touchOnBlur={true}
  touchOnChange={false}
  validate={[Function]}
/>
`;
